### PR TITLE
Add System.Runtime.InteropServices wrapper tests

### PR DESCRIPTION
**Pull request by hughbe**
These tests aren't too exciting, but they bump up the numbers for coverage

### CODE REVIEW

File: src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/MarshalTests.cs [hunk 1]
```diff
@@ -298,10 +298,10 @@ public static void GetComObjectData()
         [Fact]
         public static void GetHINSTANCE()
         {
-             Assert.Throws<ArgumentNullException>(() => Marshal.GetHINSTANCE(null));
-             IntPtr ptr = Marshal.GetHINSTANCE(typeof(int).Module);
-             IntPtr ptr2 = Marshal.GetHINSTANCE(typeof(string).Module);
-             Assert.Equal(ptr, ptr2);
+            AssertExtensions.Throws<ArgumentNullException>("m", () => Marshal.GetHINSTANCE(null));
```
Comment (danmosemsft) [inline src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/MarshalTests.cs]: Maybe we should just bulk edit Assert.Throws in the tree to be AssertExtensions..

Comment (hughbe) [inline src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/MarshalTests.cs]: Let me see what I can come up with - I'll submit a PR

**Comment by stephentoub** [comment]
@dotnet-bot test Linux x64 Release Build please

**Comment by hughbe** [comment]
Test Linux x64 Release Build

**Comment by stephentoub** [comment]
@dotnet-bot test Linux x64 Release Build please

**Comment by hughbe** [comment]
Test Linux x64 Release Build

**Comment by stephentoub** [comment]
Hugh, what was the failure last time?

**Comment by hughbe** [comment]
a test in System.Net.WebProxy

**Pull request merged**